scripts/xtensa-build-all.sh

```diff
@@ -30,6 +30,7 @@ usage: xtensa-build.sh [options] platform(s)
        -u Force UP ARCH
        -d Enable debug build
        -c Interactive menuconfig
+       -o override config
```

Comment:
This `override config` feature seems unique to SOF so it could really use a little bit more documentation. Maybe something like:
```
-o override some _defconfig values with one .config file from the src/arch/xtensa/configs/override/ directory
```